configdialog/lxqtpageselectwidget.cpp

```diff
@@ -109,7 +109,7 @@ PageSelectWidget::PageSelectWidget(QWidget *parent) :
     setSpacing(2);
     setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Expanding);
     setWordWrap(true);
-    setDragEnabled(NoDragDrop);
+    setDragEnabled(NoDragDrop != 0u);
```

Comment:
Here, you've discovered another mistake in the code (this time, not by me). `NoDragDrop` is actually `QAbstractItemView::NoDragDrop`, which is just `0`. IMHO, there's no reason to use it in that way. Instead, we could write:

```c++
setDragEnabled(false);
```